code/modules/clothing/glasses/glasses.dm

```diff
@@ -226,6 +225,30 @@
 		"Vox" = 'icons/mob/species/vox/eyes.dmi'
 		)
 
+///DELETE THIS LATER! IT SAVES MAP CODE!/////

```

Comment:
Looks like the shuttles PR got merged earlier, you're going to have to do a few map changes it would seem.